composer.json

```diff
@@ -20,8 +20,10 @@
         "symfony/dependency-injection": "^3.4|^4.0",
         "symfony/filesystem": "^3.4|^4.0",
         "symfony/finder": "^3.4|^4.0",
+        "symfony/form": "^3.4|^4.1",
         "symfony/framework-bundle": "^3.4|^4.0",
-        "symfony/http-kernel": "^3.4|^4.0"
+        "symfony/http-kernel": "^3.4|^4.0",
+        "symfony/security": "^3.4|^4.1"
```

Comment:
`symfony/security` and `symfony/form` shouldn't be dependencies. What did you need them for?